extensions/csv/src/main/java/io/deephaven/csv/CsvSpecs.java

```diff
@@ -247,12 +262,13 @@ public Charset charset() {
         return StandardCharsets.UTF_8;
     }
 
-    private CSVFormat format() {
-        return CSVFormat.DEFAULT
-                .withIgnoreSurroundingSpaces(ignoreSurroundingSpaces())
-                .withDelimiter(delimiter())
-                .withQuote(quote())
-                .withTrim(trim());
+    /**
+     * Should the CSVReader run asynchronously for better performance.
```

Comment:
I assume this implies using more than one thread potentially?  If so does it make sense to say it explicitly?

Reply:
Yeah I was using "async" to sound smart but maybe it's the wrong term? Comment changed.